src/Flash.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const BOOL_PROPS_TO_CLASS_NAMES = {
+  success: ['rev-Flash--success'],
+  warning: ['rev-Flash--warning'],
+  alert: ['rev-Flash--alert'],
+  persistent: ['rev-Flash--persistent'],
+  fade: ['rev-Flash--fade'],
+  dismissible: ['rev-Flash--dismissible'],
+}
+const BOOL_PROPS = [
+  ...Object.keys(BOOL_PROPS_TO_CLASS_NAMES),
+  'dismissible',
```

Comment:
these two are props as well right? they need to be added to the props table as well then